main/zoe/guide/contracts/covered-call.md

```diff
@@ -36,15 +36,14 @@ const aliceProposal = harden({
 
 const alicePayments = { UnderlyingAsset: aliceMoolaPayment };
 
-  // Alice redeems her invite for a seat
-const { seat: aliceSeat, payout: alicePayoutP } = await zoe.redeem(
+  // Alice makes an offer and gets an option as the outcome
+const { outcome: optionP, payout: alicePayoutP } = await E(zoe).offer(
   aliceInvite,
   aliceProposal,
   alicePayments,
 );
 
-// Alice uses her seat to create an invite for the call option
-const option = aliceSeat.makeCallOption();
+const option = await optionP;
```

Comment:
Is this copied from real code?  Since the prior code doesn't have an await, I presume this was never actually executed?  Do we need a bug to do that eventually?

Reply:
This was copied from synchronous code (I know) so there wasn't the need for an await. It executed fine